database/youtube/web3-metaverse.json

```diff
@@ -33,12 +32,19 @@
         "language": "english"
       },
      {
-
         "name": "Francesco Cuilla",
         "description": "Francesco discusses a wide range of tech topics such as DevRel, Docker, Web3, and Blockchain.",
         "url": "https://www.youtube.com/@francescociulla",
         "category": "youtube",
         "subcategory": "web3-metaverse",
         "language": "english"
+      },
+     {
+        "name": "LearnWeb3 DAO",
+        "description": "A free program that will turn you into a web3 developer.",
```

Comment:
Hey Trecia, can we have a little longer desc?

Reply:
Done ✔